.github/workflows/main.yaml

```diff
@@ -1,9 +1,12 @@
 name: Workflow for Jenkins Client
 
 on:
-  - push
-  - pull_request
-  - release
+  push:
+    branches:
+      - main
```

Comment:
IMO, we could trigger `lint` and `test` workflows in every push instead of `main` branch only.

Reply:
We don't have other branches for now.